source/frontend/src/features/leases/add/LeaseDetailSubForm.tsx

```diff
@@ -17,8 +20,71 @@ export const LeaseDetailSubForm: React.FunctionComponent<ILeaseDetailsSubFormPro
   formikProps,
 }) => {
   const { getOptionsByType } = useLookupCodeHelpers();
+
+  const { values, setFieldValue } = formikProps;
+  const { statusTypeCode, terminationReason, cancellationReason } = values;
+
+  const [currentlLeaseStatus, setLeaseStatus] = useState<string>(statusTypeCode);
```

Comment:
Also please try to avoid using state unless you are in a container component. In this case, I don't think you need state at all do you? if you need the currentLeaseStatus you can get that from the form. you can also get the initial lease status from the form.